tests/test_e2e.py

```diff
@@ -0,0 +1,109 @@
+"""End-to-end tests for Aviatrix HA module.
+
+Prerequisites for running the test:
+1. Set the following environment variables:
+    TF_VAR_customer_id: Customer ID for the Aviatrix controller
+2. If testing a non-released version of the module, set the following environment variables:
+    TF_VAR_controller_version: The version of the controller to deploy
+3. Build the dev version of the CloudFormation script, and deploy the dev lambda:
+    make push_dev
+
+What does this test do:
+1. Deploys the controller using the Terraform module
+2. Sets up controller HA using CloudFormation
+3. Terminates the running controller to trigger HA failover
+4. Verifies that the new controller is up and running by logging in to the
+   controller. The login will only succeed if backup/restore succeeds.
+"""
+import datetime
+import logging
+import os
+import time
+
+import boto3
+import pytest
+from pytest_terraform import terraform
+
+
+from aviatrix_ha.api import client
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+
+@terraform("e2e_controller", scope="session")
+@pytest.mark.skipif(os.environ.get("TF_VAR_customer_id") is None, reason="TF_VAR_customer_id is not set")
+def test_e2e_controller(e2e_controller):
```

Comment:
nice test.

Reply:
thanks! i got sick of clicking through many different UIs to do the test...